README.md

```diff
@@ -193,6 +193,14 @@ Example:
 $ python sumo_mgmt.py -url https://api.sumologic.com/api/v1/ -accessid [YOUR ACCESS ID] -accesskey [YOUR ACCESS KEY] -deleteOfflineCollectors [aliveBeforeDays]
 ```
 
+### Assign Ingest Budget to Collectors
```

Comment:
can we also add `-budgetId` to the list of capabilities under `This Python script allows managing a set of installed Sumo Logic Collectors by applying a bulk action:` above?  Maybe add

- Assigning collectors to a budget (`-assignBudget`)

Reply:
Done